MREUnityRuntime/MREUnityRuntimeLib/Assets/AssetLoader.cs

```diff
@@ -355,12 +355,19 @@ internal async void OnCreateAsset(CreateAsset payload, Action onCompleteCallback
 			// create textures
 			else if (unityAsset == null && def.Texture != null)
 			{
-				var result = await AssetFetcher<UnityEngine.Texture>.LoadTask(_owner, new Uri(def.Texture.Value.Uri));
-				unityAsset = result.Asset;
+				var result = await AssetFetcher<UnityEngine.Texture2D>.LoadTask(_owner, new Uri(def.Texture.Value.Uri));
 				if (result.FailureMessage != null)
 				{
 					response.FailureMessage = result.FailureMessage;
 				}
+				else
+				{
+					var wwwTexture = result.Asset;
+					var mipTexture = new Texture2D(wwwTexture.width, wwwTexture.height, wwwTexture.format, true, false);
+					Graphics.CopyTexture(wwwTexture, 0, 0, mipTexture, 0, 0);
+					mipTexture.Apply(true);
+					unityAsset = mipTexture;
+				}
```

Comment:
Textures are ~90% of an MRE's memory footprint. I don't think we have enough margin here to increase the size of all textures by 33%. We're already short on GPU mem without this. That said, mipmaps are super useful. I think this should be exposed to the SDK, defaulting to false, so the developer can decide.